manubot/pandoc/cite_filter.py

```diff
@@ -154,19 +162,32 @@ def process_citations(doc: pf.Doc) -> None:
 
     - bibliography (use to define reference metadata manually)
     - citekey-aliases (use to define tags for cite-by-id citations)
+    - manubot-bibliography-cache:
```

Comment:
To make sure I'm following the usage correctly, to enable the desired caching would a user provide the same path to `manubot-bibliography-cache` and `manubot-output-bibliography`?  Or they could use different paths, but populate `manubot-bibliography-cache` by taking `manubot-output-bibliography` as a starting point and manually deleting any entries that should not be cached?